### ISSUE TITLE
finder app banner text is not responsive

### ISSUE THREAD

**Issue by cmenedes** [timkeane/nyc-lib#10]
## Expected Behavior
Finder App banner text should be responsive. The text should follow the size of the browser window while following AA/AAA guidelines for accessibility (font-size, contrast, etc). See https://www.w3.org/TR/WCAG21/ for accessibility guidelines. The text should be accessible to those on both larger and smaller devices.

## Current Behavior
Finder App banner text cuts itself off due to the margin created from the "back to map" button (and because it isn't designed to be responsive). On smaller devices/screens, the full text is simply not visible in its entirety. See screenshots below in example app created for DOHMH:

![Visible banner text on large screen](https://user-images.githubusercontent.com/11576888/67022818-d4c3a980-f0cf-11e9-86ee-bba0d10f3d33.png)
![Invisible banner text on large screen](https://user-images.githubusercontent.com/11576888/67022937-fde43a00-f0cf-11e9-9338-7a36cb424ce9.png)
![Partially visible banner text on large screen](https://user-images.githubusercontent.com/11576888/67022984-0d638300-f0d0-11e9-85fd-a8dcc00eda04.png)

### Steps to Reproduce
Build nyc-lib as described in readme
Open finderapp.html located in the dist/examples/ol folder on either a mobile device or by resizing from a larger screen.
Inspect banner text

**Comment by cmenedes** [comment]
Finalized in e1e3e73
